src/_P145_MQxxx.ino

```diff
@@ -175,10 +175,13 @@ boolean Plugin_145(byte function, struct EventStruct *event, String& string)
       bool compensate = P145_PCONFIG_FLAGS & 0x0001;
       bool calibrate = (P145_PCONFIG_FLAGS >> 1) & 0x0001;
       addFormCheckBox(F("Enable automatic calibration"), F("plugin_145_enable_calibrarion"), calibrate);
-      addFormCheckBox(F("Enable temp/humid compensation"), F("plugin_145_enable_compensation"), compensate);
-      addFormNote(F("If this is enabled, the Temperature and Humidity values below need to be configured."));
-      //if (compensate)
+      //addFormCheckBox(F("Enable temp/humid compensation"), F("plugin_145_enable_compensation"), compensate);
+      addFormSelector_YesNo(F("Enable temp/humid compensation"), F("plugin_145_enable_compensation"), compensate, true);
```

Comment:
To save some .bin size, please use short values for the id's, so instead of `plugin_145_enable_calibration` you could use `en_cal`, saving 24 * 2 = 48 bytes just in this single id... many of these smalls become 1 big saving 😃